tests/template_tests/test_autoescape_setting.py

```diff
@@ -0,0 +1,22 @@
+
+
+from django.template import EngineHandler
+from django.test import SimpleTestCase
+
+
+class AutoescapeSettingTest(SimpleTestCase):
+
+    """ Test for autoescape setting for DjangoTemplates, #25469 """
+
+    def test_autoescape_off(self):
+        templates = [{'BACKEND': 'django.template.backends.django.DjangoTemplates',
+                      'OPTIONS': {'autoescape': False}}]
+        engines = EngineHandler(templates=templates)
+        self.assertEqual(engines['django'].from_string('Hello, {{ name }}').render({'name': 'Bob & Jim'}),
```

Comment:
Please use 4 space indent as done in other `self.assertEqual` assertions in the file I mentioned.